website/docs/guides/best-practices/materializations/materializations-guide-6-examining-builds.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Examining our builds"
+id: materializations-guide-6-examining-builds
+description: Learn how to utilize materializations in dbt.
+displayText: Materializations best practices
+hoverSnippet: Learn how to utilize materializations in dbt.
+---
+
+## Examining our builds
+
+- ⌚ dbt keeps track of how **long each model took to build**, when it started, when it finished, its completion status (error, warn, or success), its materialization type, and _much_ more.
+- 🖼️ This information is stored in a couple files which dbt calls **artifacts**.
+- 📊 Artifacts contain a ton of information in JSON format, so aren’t easy to read, but **dbt Cloud** packages the most useful bits of information into a tidy **visualization** for you.
+- ☁️ If you’re not using Cloud, we can still use the output of the **dbt CLI to understand our runs**.
+
+### Model Timing
+
+That’s where dbt Cloud’s Model Timing visualization comes in extremely handy. If we’ve set up a [Job](https://docs.getdbt.com/docs/get-started/getting-started/building-your-first-project/schedule-a-job#create-and-run-a-job) in dbt Cloud to run our models, we can use the Model Timing tab to pinpoint our longest-running models.
+
+![dbt Cloud's Model Timing diagram](/img/guides/best-practices/materializations/model-timing-diagram.png)
+
+- 🧵 This view lets us see our **mapped out in threads** (up to 16 threads, we’re currently running with 4, so we get 4 tracks) over time. You can think of **each thread as a lane on a highway**.
```

Comment:
This is now able to be up to 64 threads! https://dbt-labs.slack.com/archives/C017GDLAF7D/p1673567967628509?thread_ts=1673476115.653289&cid=C017GDLAF7D